test/unit/spec/v2/view-builder/internals/FormInputFactory_spec.js

```diff
@@ -0,0 +1,82 @@
+import { Collection } from 'okta';
+import FactorEnrollOptions from 'v2/view-builder/components/FactorEnrollOptions';
+import FormInputFactory from 'v2/view-builder/internals/FormInputFactory';
+
+describe('v2/view-builder/internals/FormInputFactory', function () {
+  it('handles text type', function () {
+    const opt = {
+      type: 'text'
+    };
+    expect(FormInputFactory.create(opt)).toEqual({
+      type: 'text',
+      'label-top': true,
+    });
+    expect(opt).toEqual({
```

Comment:
This expectation seems redundant ?

Reply:
sort of. trying to make sure `opt` is not mutated.